ocfweb/docs/docs/staff/scripts/sorry.md

```diff
@@ -0,0 +1,28 @@
+[[!meta title="sorry: ban/disable an OCF account"]]
+
+## Introduction
+
+`sorry` is the command used to ban or disable OCF accounts. Accounts are banned 
+for various reasons (users/groups being disruptive or otherwise violating 
+OCF/university policies, security issues, lack of contact information, etc.) 
+and alumni sometimes request to have their accounts disabled to stop vhosts, 
+take down information, or for myriad other reasons.
+
+## Usage/Example
+
+    Usage: sorry [user to be sorried] [sorry file]
```

Comment:
If I'm not mistaken, the sorry file is copied to ~user/.sorry, right? I think I mention that in "copies the sorry file... to ~user/.sorry"